toolkit/Makefile

```diff
@@ -25,6 +25,8 @@ PACKAGE_REBUILD_LIST    ?=
 PACKAGE_IGNORE_LIST     ?=
 ##help:var:SRPM_PACK_LIST:<spec_list>=List of space-separated spec folders inside "SPECS_DIR" to analyze for the build. If empty, all items inside the "SPECS_DIR" will be analyzed. Example: SRPM_PACK_LIST="kernel go which".
 SRPM_PACK_LIST          ?=
+##help:var:SRPM_DOWNLOAD_MODE:<mode>=Mode to use for downloading SRPMs. Modes are define in the SRPM packager functionality.
+SRPM_DOWNLOAD_MODE      ?=
```

Comment:
This feels more closely connected with `SOURCE_URL` further down in the file. What do you think about re-naming it to `SOURCE_AUTH_MODE`?